### PR TITLE
option to install mixedcontent

### PR DESCRIPTION
**Pull request by Carreau**
Should fix readme too.

### CODE REVIEW

File: jupyterdrive/__init__.py [hunk 1]
```diff
@@ -57,7 +58,13 @@ def main(argv=None):
...
+    parser.add_argument("-m", "--mixed", help="Installed the mixed content manager",
+                    action="store_true")
+    parser.add_argument("-S", "--no-symlink", help="do not symlink at install time",
+                    action="store_false")
     args = parser.parse_args(argv)
-    install(profile=args.profile)
-
+    if not args.profile:
+        parser.print_help()
+    sys.exit(0)
```
Comment (KesterTong) [inline jupyterdrive/__init__.py]: add four spaces :-)

**Comment by KesterTong** [comment]
I get this error
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/google/home/kestert/jupyter-drive/jupyterdrive/__main__.py", line 2, in <module>
    jupyterdrive.main()
  File "jupyterdrive/__init__.py", line 74, in main
    install(profile=args.profile, mixed=args.mixed, symlink=args.no_symlink)
  File "jupyterdrive/__init__.py", line 20, in install
    nbe.install_nbextension(os.path.join(dname,'gdrive'), symlink=symlink)
  File "/usr/local/google/home/kestert/ipython/IPython/html/nbextensions.py", line 176, in install_nbextension
    ensure_dir_exists(nbext)
  File "/usr/local/google/home/kestert/ipython/IPython/utils/path.py", line 588, in ensure_dir_exists
    os.makedirs(path, mode=mode)
  File "/usr/local/google/home/kestert/ip/lib/python2.7/os.py", line 150, in makedirs
    makedirs(head, mode)
  File "/usr/local/google/home/kestert/ip/lib/python2.7/os.py", line 157, in makedirs
    mkdir(name, mode)
OSError: [Errno 13] Permission denied: '/usr/local/share/jupyter'
```
But when I create a profile it goes to ```~/.ipython``` not ```/usr/local/share/jupyter```

File: jupyterdrive/__init__.py (hunk 1 above)
Comment (Carreau) [inline jupyterdrive/__init__.py]: :-P

**Comment by Carreau** [comment]
> But when I create a profile it goes to ~/.ipython not /usr/local/share/jupyter

Hum... weird.

**Comment by Carreau** [comment]
Updated. could you retry and pass it the `--user` flag ? Did you install IPython through apt ? or as root ?

**Comment by KesterTong** [comment]
I'm using virtual env.  I think that means it should be installed as user.  Is that right?

File: jupyterdrive/__init__.py [hunk 2]
```diff
@@ -57,7 +60,17 @@ def main(argv=None):
...
+                    action="store_true")
     args = parser.parse_args(argv)
-    install(profile=args.profile)
-
+    if not args.profile:
+        parser.print_help()
+        sys.exit(0)
+    install(profile=args.profile,
+              mixed=args.mixed,
+            symlink=args.no_symlink)
```
Comment (KesterTong) [inline jupyterdrive/__init__.py]: should be ```not args.no_symlink```.

Comment (KesterTong) [inline jupyterdrive/__init__.py]: ignore that, I didn't understand how command line params were defined

Comment (Carreau) [inline jupyterdrive/__init__.py]: Should it ? cause above it's `store_false` ? I'll re-check. I can try set metavar to be more readable.

**Comment by KesterTong** [comment]
I get the same problems with HEAD, and so I'm just going to merge these and create an issue.  It works with my laptop.

File: jupyterdrive/__init__.py (hunk 2 above)
Comment (KesterTong) [inline jupyterdrive/__init__.py]: no I was wrong.  Don't worry about changing it.

**Pull request merged**